src/main/scala/com/gu/rcspollerlambda/services/HTTP.scala

```diff
@@ -20,12 +20,18 @@ object HTTP extends Config {
 
   def getXml(lastid: String): Either[LambdaError, String] = {
     logger.info(s"Fetching XML from $rcsUrl?lastid=$lastid&subscribername=TEST")
-    Await.result(wsClient.url(rcsUrl).withQueryStringParameters(("lastid", lastid), ("subscribername", "TEST")).get().map { result =>
-      logger.info(s"Result of GET request was ${result.status}")
-      result.status match {
-        case 200 => Right(result.body)
-        case status => Left(RCSError(status, result.body))
-      }
-    }, 120.seconds)
+    try {
+      Await.result(wsClient.url(rcsUrl).withQueryStringParameters(("lastid", lastid), ("subscribername", "TEST")).get().map { result =>
+        logger.info(s"Status of GET request was ${result.status}")
+        result.status match {
+          case 200 => Right(result.body)
+          case _ => Left(RCSError(result.body))
+        }
+      }, 120.seconds)
```

Comment:
as the lambda now waits for 5 minutes, should this timeout be increased too?

Reply:
Yes, let's wait 4 minutes then 👍